docs/snippets/defaults_channel_mapping.mdx

```diff
@@ -27,15 +27,15 @@ default_channel_by_device:
 default_device_by_channel:
   social:   phone
   ctv-bvod: tv
-  streaming-video: tv
+  streaming-video: pc
```

Comment:
@pymble2073 Can we please move this change to a follow up PR. We need to sequence this change because of a necessary update to our backend to prevent this new default taking effect immediately